microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md

```diff
@@ -112,15 +112,11 @@ Once you have a recording of the page's performance, you can assess the page's p
 
 The main metric for measuring the performance of any animation is frames per second (FPS).  Users are happy when animations run at 60 FPS.
```

Comment:
I think this should be cut and the `### Analyze frames per second` section title. We don't have FPS tracked any more in the Performance tool so everything related to FPS needs to be removed

Reply:
done